### PR TITLE
update nuget to 3.5.0-beta2-1392

### PR DESCRIPTION
**Pull request by rohit21agrawal**
Update nuget to 3.5.0-beta2-1392 . This build has been upgraded to depend on the following:
•	CoreCLR/CoreFX/WCF – rc3-24128-00
•	Microsoft.NetCore.App – 1.0.0-rc3-004338
•	CLI – 1.0.0-preview2-002911

### CODE REVIEW

**Comment by joshfree** [comment]
@brthor @eerhardt @livarcocc could you assist with getting the latest NuGet client into /dotnet/cli

**Comment by brthor** [comment]
@rohit21agrawal Please retarget this PR to branch `rel/1.0.0-preview2`

> System.TimeoutException : Response time out after 10 seconds.

@troydai These DTH tests are timing out all over the place

> 17:16:50 Failed to resolve full path of the current executable [/proc/curproc/exe]

@schellap @livarcocc We fixed this in preview2 right?

**Comment by brthor** [comment]
@rohit21agrawal We are having some issues with ubuntu builds at the moment PRs #3331 and #3334 are going in soon to fix this. 

We can retry the other builds. These DTH test timeouts have just started appearing and we haven't figured out why yet.

File: build_projects/dotnet-cli-build/project.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "System.Runtime.Serialization.Primitives": "4.1.1-rc3-24131-00",
     "System.Xml.XmlSerializer": "4.0.11-rc3-24131-00",
     "WindowsAzure.Storage": "6.2.2-preview",
-    "NuGet.CommandLine.XPlat": "3.5.0-rc-1285",
+    "NuGet.CommandLine.XPlat": "3.5.0-beta2-1392",
```
Comment (eerhardt) [inline build_projects/dotnet-cli-build/project.json]: Isn't this a downgrade?

**Comment by livarcocc** [comment]
> 17:16:50 Failed to resolve full path of the current executable [/proc/curproc/exe]
@schellap Can you take a look? For me, this was inconsistent. I am not sure why that happens, but @schellap needs to look into it.

**Comment by schellap** [comment]
@dotnet-bot test Debian8.2 x64 Debug Build please

Firing a run to log on to the CI machine.

**Comment by schellap** [comment]
@dotnet-bot test Debian8.2 x64 Debug Build please

File: build_projects/dotnet-cli-build/project.json (hunk 1 above)
Comment (rohit21agrawal) [inline build_projects/dotnet-cli-build/project.json]: As far as I know, we made a mistake in the build system where beta packages published to the myget feed were labelled as RC.

@rrelyea  might have more context on this.

**Comment by rohit21agrawal** [comment]
@brthor @eerhardt @livarcocc 

have retargeted the pull request to rel/1.0.0-preview2

https://github.com/dotnet/cli/pull/3337

**Pull request closed without merging**